ldk/javascript/examples/self-test-loop/src/utils/string.ts

```diff
@@ -0,0 +1,19 @@
+export enum StringOptions {
+  IgnoreCase,
+}
+
+// TODO: Required as windows filesystem create converts file path to lower case. Remove after sidekick is fixed. Ian is working on it.
```

Comment:
Is there a Jira card number? This comment is probably more relevant on the test that invokes `areStringsEqual`

Reply:
Updated.